### ISSUE TITLE
nested FormData 

### ISSUE THREAD

**Issue by avlnche64** [acacode/swagger-typescript-api#262]
HttpClient has FormData formatter, which transforms request body to FormData object if ContentType set to 'multipart/form-data'
The problem is that formatter can't handle nested objects in body. 
For example, if you have such object
```
{
    foo: '123',
    bar: {
        a: 'a',
        b: 'b'
    }
}
```
formatter will return such FormData
```
foo: '123'
bar: '[object Object]'
```

**Comment by js2me** [comment]
Hello @avlnche64 !   
Thanks for this issue
It will be fixed in next release

**Comment by js2me** [comment]
Should be fixed in 9.1.1
